internal/conversation/whatsapp.go

```
@@ -0,0 +1,323 @@
+	if windowOpen {
+		content := m.i18n.Ts("conversation.whatsapp.csatMessage", "link", csatURL)
+		if _, err := m.QueueReply(nil, conversation.InboxID, actorUserID, conversation.ContactID, conversation.UUID, content, nil, nil, nil, meta); err != nil {
+			m.lo.Error("error sending whatsapp CSAT link", "conversation_uuid", conversation.UUID, "error", err)
+			return envelope.NewError(envelope.GeneralError, m.i18n.T("globals.messages.somethingWentWrong"), nil)
+		}
+		return nil
+	}
+
+	actor, err := m.userStore.GetSystemUser()
+	if err != nil {
+		return nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 167) but it returns nil

(nilerr)